File: patterns/toggle-pattern/rules-js/toggle-pattern-feedback.js
rules.JSRule({
  name: "Toggle Lampe mit Feedback",
  triggers: [triggers.ItemCommandTrigger("mqttSwitchIn2", "OFF")],
  execute: (event) => {
    const lamp = items.getItem("modbusSwitchOut1");
    if (lamp.state !== "ON") {
      lamp.sendCommand("ON");
    } else {
      lamp.sendCommand("OFF");
    }
    lamp.postUpdate(lamp.state);
  }
});

File: patterns/toggle-pattern/rules-python/toggle_pattern_feedback.py
from openhab import rule, Registry
from openhab.triggers import ItemCommandTrigger


@rule(triggers=[ItemCommandTrigger("mqttSwitchIn2", "OFF")])
class ToggleLampeMitFeedback:
    def execute(self, module, input):
        lamp = Registry.getItem("modbusSwitchOut1")
        if str(lamp.getState()) != "ON":
            lamp.sendCommand("ON")
        else:
            lamp.sendCommand("OFF")
        lamp.postUpdate(str(lamp.getState()))
